introduction.tex

```diff
@@ -1,8 +1,18 @@
 \section{Introduction}
 % Convince the reader that Distributed Machine Learning is significant / important / interesting
-% Convince the reader that we shouldn't be completely satisfied with the existing literature on this topic and that our research will address some imiportant limitation or deficiency
+% Convince the reader that we shouldn't be completely satisfied with the existing literature on this topic and that our research will address some important limitation or deficiency
 % Explain our
 	% RESEARCH QUESTION
 	% SUB-RESEARCH QUESTIONS
 	% ALL OF THEM WELL-MOTIVATED
 % Give a hypothesis for them
+
+% Clearly formulated andwell-motivated research question. Main research question split up into sub-questions.
+% Very well-structured document. General presentation of the content (text and figures) is very effective. 
+% Excellent expressed and formulated report. Document has a smooth flow. Spelling and grammatically error free.
+
+With the ever increasing amount of data created every day and the prevalence of Machine Learning (ML) algorithms in analyzing all this data, a lot of research is being done on the application of Distributed Machine Learning approaches. Because of the many different approaches to dealing with so-called Big Data, we would like to give an overview of the current state-of-the-art of Distributed Machine Learning.
+
+Most Machine Learning algorithms assume that all data can be easily accessed. This is not always the case, as some forms of data are inherently distributed, like transaction information for multinationals, or otherwise too big to store on single machines, like astronomical data. Regular Machine Learning algorithms have to be changed to accommodate for these circumstances to still be able to extract useful information from these datasets.
```

Comment:
First sentence could be split into about three shorter ones. Final sentence could be less structurally repetitive.